README.md

```diff
@@ -132,6 +132,27 @@ This action will download an Augmented SBOM file in `$RELEASE_ASSETS/sbom.json`.
     silk_asset_group: mongodb-python-driver
 ```
 
+## Upload S3 assets
+
+A number of scripts create files in the `tmp/s3_assets` folder, which then can
+be uploaded to the product's S3 bucket:
+
+```yaml
+- name: Setup
+  uses: mongodb-labs/drivers-github-tools/setup@v2
+  with:
+    ...
+
+- name: Upload S3 assets
+  uses: mongodb-labs/drivers-github-tools/upload-s3-assets@v2
+  with:
+    version: <release version>
+    product_name: <product_name>
+```
+
+Optionally, you can specify which files to upload using the `filenames` input.
+By default, all files in the S3 directory are uploaded.
```

Comment:
Why not support `dry_run` and print out the files that would have been uploaded if it is set?

Reply:
I didn't think it was necessary, but I can see how it can be useful during the initial phase of setting up the tooling. I've added the `dry_run` parameter with slightly changed logic: it prints the filename along with the name it would receive in the bucket (minus the bucket name since we'd be relying on it being redacted) and doesn't `cat` the file to avoid exposing data that should be kept internal.